src/lib/adapters/waku/index.ts

```diff
@@ -140,6 +141,7 @@ async function executeOnDataMessage(
 		const users = chat ? chat.users : []
 		const args: WakuObjectArgs = {
 			...context,
+			chatName: chat?.name ?? 'Unknown', // FIXME: figure out how to have a proper chatname
```

Comment:
`chatName` is either `chat.name` (for group chats) or it is the name of person you talk to in a private chat (that is `users[0].name` by convention)

Reply:
Oh nice, can I rely on the `[0]` index? Or should I use `users.find(u => u.address !== profile.address)?.name`

Reply:
Currently we rely on the `[0]` index elsewhere in the code, but that is not really documented anywhere and it's not enforced either, so it may be better to use the `find` version.

It would be nice to add this to a utility function in the chat store, so if we change it later then we only need to change it at one place. This can be done in a separate PR though.